apps/opik-backend/src/test/java/com/comet/opik/infrastructure/http/cors/CorsDisabledE2ETest.java

```diff
@@ -0,0 +1,73 @@
+package com.comet.opik.infrastructure.http.cors;
+
+import com.comet.opik.api.resources.utils.ClickHouseContainerUtils;
+import com.comet.opik.api.resources.utils.ClientSupportUtils;
+import com.comet.opik.api.resources.utils.MySQLContainerUtils;
+import com.comet.opik.api.resources.utils.RedisContainerUtils;
+import com.comet.opik.api.resources.utils.TestDropwizardAppExtensionUtils;
+import com.redis.testcontainers.RedisContainer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.testcontainers.clickhouse.ClickHouseContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import ru.vyarus.dropwizard.guice.test.ClientSupport;
+import ru.vyarus.dropwizard.guice.test.jupiter.ext.TestDropwizardAppExtension;
+
+import static com.comet.opik.api.resources.utils.ClickHouseContainerUtils.DATABASE_NAME;
+
+@Testcontainers(parallel = true)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("CORS headers logic test when CORS config is disabled")
+class CorsDisabledE2ETest {
+    private static final RedisContainer REDIS = RedisContainerUtils.newRedisContainer();
+
+    private static final MySQLContainer<?> MYSQL = MySQLContainerUtils.newMySQLContainer();
+
+    private static final ClickHouseContainer CLICKHOUSE = ClickHouseContainerUtils.newClickHouseContainer();
+
+    @RegisterExtension
+    private static final TestDropwizardAppExtension app;
+
+    static {
+        MYSQL.start();
+        CLICKHOUSE.start();
+        REDIS.start();
+
+        var databaseAnalyticsFactory = ClickHouseContainerUtils.newDatabaseAnalyticsFactory(
+                CLICKHOUSE, DATABASE_NAME);
+
+        app = TestDropwizardAppExtensionUtils.newTestDropwizardAppExtension(
+                MYSQL.getJdbcUrl(),
+                databaseAnalyticsFactory,
+                REDIS.getRedisURI(),
+                false);
+    }
+
+    private String baseURI;
+    private ClientSupport client;
+
+    @BeforeAll
+    void setUpAll(ClientSupport client) {
+
+        this.baseURI = "http://localhost:%d".formatted(client.getPort());
+        this.client = client;
+
+        ClientSupportUtils.config(client);
+    }
+
+    @Test
+    @DisplayName("Should not contain CORS headers")
+    void testCorsDisabled() {
+        var response = client.target("%s/is-alive/ping".formatted(baseURI))
+                .request().header("Origin", "localhost")
+                .get();
+
+        Assertions.assertEquals(200, response.getStatus());
+        Assertions.assertFalse(response.getHeaders().containsKey("Access-Control-Allow-Origin"));
```

Comment:
You should use assertJ assertions in this project instead of JUnit assertions.

Reply:
Done in https://github.com/comet-ml/opik/pull/498/commits/1c7cd0b6d75638026a8dbae0dee2c3722a214c7e